### PR TITLE
Add missing env setup step

### PR DESCRIPTION
**Pull request by Callisto13**
### Description

Closes https://github.com/weaveworks/profiles/issues/235.

### CODE REVIEW

**Review by Himangini — APPROVED** [review]

File: userdocs/profiles.dev/docs/tutorial-basics/setup.md [hunk 1]
```diff
@@ -42,16 +42,34 @@ version regularly updated.
...
+Profiles relies on Flux to deploy artifacts to your cluster, this means that at a minimum
+you much have the following Flux CRDs installed:
 
-To install both the profiles CRD and the required Flux CRDs, run:
+- [`helmreleases.helm.toolkit.fluxcd.io`](https://github.com/fluxcd/helm-controller/blob/main/config/crd/bases/helm.toolkit.fluxcd.io_helmreleases.yaml)
+- [`gitrepositories.source.toolkit.fluxcd.io`](https://github.com/fluxcd/source-controller/blob/main/config/crd/bases/source.toolkit.fluxcd.io_gitrepositories.yaml)
+- [`helmrepositories.source.toolkit.fluxcd.io`](https://github.com/fluxcd/source-controller/blob/main/config/crd/bases/source.toolkit.fluxcd.io_helmrepositories.yaml)
+- [`kustomizations.kustomize.toolkit.fluxcd.io`](https://github.com/fluxcd/kustomize-controller/blob/main/config/crd/bases/kustomize.toolkit.fluxcd.io_kustomizations.yaml)
+
+You can either install these manually by downloading and applying the above linked CRD
```
Comment (aclevername) [inline userdocs/profiles.dev/docs/tutorial-basics/setup.md:53]: those are just the CRDs, not the controller which we always need. I don't know why we'd document any way to install other than `flux install` :thinking: ?

Comment (Callisto13) [reply userdocs/profiles.dev/docs/tutorial-basics/setup.md:53]: yep, was not sure, will delete

File: userdocs/profiles.dev/docs/tutorial-basics/setup.md [hunk 2]
```diff
@@ -42,16 +42,33 @@ version regularly updated.
...
+Or to install specific components:
+
+```bash
+flux install --components="source-controller,kustomize-controller,helm-controller"
+```
+
+Next install the Profiles CRD, with:
 
 ```bash
 pctl prepare
```
Comment (Skarlso) [inline userdocs/profiles.dev/docs/tutorial-basics/setup.md:68]: Would you mind changing this to `install` as well? @aclevername It's install now, right? :D Not bootstrap. :D

**Review by Skarlso — APPROVED** [review]

File: userdocs/profiles.dev/docs/tutorial-basics/setup.md (hunk 2 above)
Comment (Callisto13) [reply userdocs/profiles.dev/docs/tutorial-basics/setup.md:68]: gah you meddling kids

**Pull request merged**